src/diffusers/loaders.py

```diff
@@ -330,13 +346,32 @@ def load_attn_procs(self, pretrained_model_name_or_path_or_dict: Union[str, Dict
                 lora_grouped_dict[attn_processor_key][sub_key] = value
 
             for key, value_dict in lora_grouped_dict.items():
-                rank = value_dict["to_k_lora.down.weight"].shape[0]
-                hidden_size = value_dict["to_k_lora.up.weight"].shape[0]
-
                 attn_processor = self
                 for sub_key in key.split("."):
                     attn_processor = getattr(attn_processor, sub_key)
 
+                # Process non-attention layers
+                if "lora.down.weight" in value_dict:
```

Comment:
Is this `lora.down.weight` check safe to disambiguate between if we're loading weights into an attention processor or not? I believe the attention processors also use the same linear layer and would have the same key present? 

If it is a safe enough check to disambiguate, a brief comment on why in the code would be really helpful

Reply:
I've tried to amend the comment by saying this is not like the attention processors where they used to `to_{k,q,v,out_proj_lora` convention. I can also try to figure out a different way to compare if this is conflicting (although i didn't see it in any of my tests) but from what i've gathered this seems pretty unique to the layers we are handling.

Reply:
I had hit the same question as @williamberman but I arrived at a similar conclusion as @isidentical:

> but from what i've gathered this seems pretty unique to the layers we are handling.

@isidentical maybe let's add a bit more elaborative comment?

Reply:
Ok, I trust if you two have looked into it and tests still pass. In general though I have a hard time following this bit. We should re-visit if it's possible to know what update to make to the model architecture without looking inside the state dict. If you can't tell, I don't like looking inside the state dict :)